src/components/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./HeroSection.scss";
+
+function HeroSection() {
+	return (
+		<div className="HeroSection">
+			<div className="HeroSection-main">
```

Comment:
the className should be "HeroSection__main"

Reply:
fixed